doc/toc.yml

```diff
@@ -1,19 +1,89 @@
 - name: Overview
-  href: Overview/ExtensionsOverviewAndGettingStarted.md
+  href: ExtensionsOverview.md
+- name: Frequently Asked Questions
+  href: Questions.md
+- name: Creating an Application
+  href: GettingStarted.md
+- name: Features
   items:
+    - name: Authentication
```

Comment:
These should be in alphabetic order - eg Hosting isn't correct

Reply:
@nickrandolph corrected in https://github.com/unoplatform/uno.extensions/pull/1089/commits/c307bf604dd56a54cae3d7177cf41713477f5585